backend/main/server/controllers.go

```diff
@@ -344,6 +351,13 @@ func (a *App) updateCommunity(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	//Validate Contract Thresholds
+	err = validateConractThreshold(*payload.Strategies)
```

Comment:
Spelling error - should be validateCon**t**ractThreshold

Reply:
nice catch